common/fdlimit/fdlimit_windows.go

```diff
@@ -14,13 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
 
-package utils
+package fdlimit
 
 import "errors"
 
-// raiseFdLimit tries to maximize the file descriptor allowance of this process
+// RaiseFdLimit tries to maximize the file descriptor allowance of this process
 // to the maximum hard-limit allowed by the OS.
-func raiseFdLimit(max uint64) error {
+func RaiseFdLimit(max uint64) error {
```

Comment:
Please rename the functions to `FdLimit` doesn't appear in the name.

Reply:
@fjl thanks for the feedback. I tried to find better names than `Raise`, `Get` and `GetMax` but couldn't come up with something better.

Any suggestions?

Reply:
@ricardohsd You could use `Current` and `Maximum`. Then you would have:

`fdlimit.Raise`, `fdlimit.Current` and `fdlimit.Maximum`.

Reply:
@karalabe thanks!